plugins/org.eclipse.xtext.ide/xtend-gen/org/eclipse/xtext/ide/editor/contentassist/IdeContentProposalPriorities.java

```diff
@@ -5,27 +5,27 @@
  * which accompanies this distribution, and is available at
  * http://www.eclipse.org/legal/epl-v10.html
  */
-package org.eclipse.xtext.web.server.contentassist;
+package org.eclipse.xtext.ide.editor.contentassist;
 
 import com.google.common.base.Objects;
 import com.google.inject.Singleton;
 import org.eclipse.xtend.lib.annotations.AccessorType;
 import org.eclipse.xtend.lib.annotations.Accessors;
+import org.eclipse.xtext.ide.editor.contentassist.ContentAssistEntry;
 import org.eclipse.xtext.resource.IEObjectDescription;
-import org.eclipse.xtext.web.server.contentassist.ContentAssistResult;
 import org.eclipse.xtext.xbase.lib.Pure;
 
 @Singleton
 @Accessors({ AccessorType.PROTECTED_GETTER, AccessorType.PROTECTED_SETTER })
 @SuppressWarnings("all")
-public class WebContentProposalPriorities {
+public class IdeContentProposalPriorities {
```

Comment:
This seems to be a good candidate to extract smth like AbstractContentProposalPriorities that can be reused from the SWT based impl. Or the constant values could delegate from there to this one.